packages/eslint-config-cypress/index.js

```diff
@@ -11,5 +11,12 @@ module.exports = {
         'cypress/no-force': 'error',
       },
     },
+    {
+      files: ['cypress.config.ts'],
```

Comment:
It might be worth making the end of the extension a `*` so that the config isn't required to be TypeScript.

Reply:
Fair point - the assumption was that all of our repos would use TS since we have it in all of the UI repos. That said, if there is an unforeseen use case it may still be nice to have it handled